docs/reference/inference/post-inference.asciidoc

```diff
@@ -99,39 +113,90 @@ The API returns the following response:
 [source,console-result]
 ------------------------------------------------------------
 {
-  "sparse_embedding": [
+  "completion": [
     {
-      "port": 2.1259406,
-      "sky": 1.7073475,
-      "color": 1.6922266,
-      "dead": 1.6247464,
-      "television": 1.3525393,
-      "above": 1.2425821,
-      "tuned": 1.1440028,
-      "colors": 1.1218185,
-      "tv": 1.0111054,
-      "ports": 1.0067928,
-      "poem": 1.0042328,
-      "channel": 0.99471164,
-      "tune": 0.96235967,
-      "scene": 0.9020516,
-      (...)
-    },
-    (...)
+      "result": "Elastic is a company that provides a range of software solutions for search, logging, security, and analytics. Their flagship product is Elasticsearch, an open-source, distributed search engine that allows users to search, analyze, and visualize large volumes of data in real-time. Elastic also offers products such as Kibana, a data visualization tool, and Logstash, a log management and pipeline tool, as well as various other tools and solutions for data analysis and management."
+    }
   ]
 }
 ------------------------------------------------------------
 // NOTCONSOLE
 
+[discrete]
+[[inference-example-rerank]]
+===== Rerank example
```

Comment:
It might be useful to reference this document somewhere in our docs https://docs.cohere.com/docs/elasticsearch-and-cohere#build-a-hybrid-search-index-with-cohere-and-elasticsearch

Reply:
I've added the link in the latest commit.